### ISSUE TITLE
Support tail -f / -F

### ISSUE THREAD

**Issue by martin-braun** [shelljs/shx#172]
`npx shx tail -f` or `npx shx tail -F` will complain that its option it's unknown. It would be nice to have cross-platform support for `tail -f / -F`.

Is this already on the roadmap?

**Comment by nfischer** [comment]
Thanks! I filed this as https://github.com/shelljs/shelljs/issues/986, since support would need to land first over there. I'll mark this ticket as a duplicate now of that one.

**Comment by martin-braun** [comment]
Thanks, yes, makes more sense.
